packages/orbit-components/src/Toast/ToastMessage.jsx

```diff
@@ -56,7 +56,6 @@ const StyledInnerWrapper = styled.div`
       width: initial;
       padding: ${theme.orbit.spaceSmall};
     `)}
-    }
```

Comment:
I wonder how this was not a problem before 🤯

Reply:
SC parses that later and fixing :)